doc-content/enterprise-only/openshift/environment-immutable-monitoring-proc.adoc

```diff
@@ -1,19 +1,23 @@
 [id='environment-immutable-monitoring-proc']
+// definitions for RH-SSO include
+:centralonly:
+// centralonly must be undefined at end of file
+:LOCAL_CENTRAL: {CENTRAL} Monitoring
 = Deploying monitoring and Smart Router for an environment with immutable servers
 
 If you want to use {CENTRAL} Monitoring and Smart Router for an environment with immutable servers, you must deploy them before deploying any {KIE_SERVERS}. If you do not want to use these components, skip this procedure.
 
-To deploy {CENTRAL} Monitoring and Smart Router for an environment with immutable servers, use the `rhpam70-prod-immutable-monitor.yaml` template file. You can extract this file from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download the file from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
+To deploy {CENTRAL} Monitoring and Smart Router for an environment with immutable servers, use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-monitor.yaml` template file. You can extract this file from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download the file from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
```

Comment:
For some reason there is a paragraph break in the rendered output that you gave me at "You can extract this file [break] from the ..."